gulp/tasks/clean.js

```diff
@@ -4,6 +4,25 @@ const configClean = require( '../config' ).clean;
 const del = require( 'del' );
 const gulp = require( 'gulp' );
 
+
+/**
+ * Cleans out built CSS files
+ */
+function cleanCSS() {
+  del( configClean.css + '/**/*' );
```

Comment:
Add a return statement here

Reply:
What should it return?

Reply:
sorry, so it should just return `del( configClean.css + '/**/*' );`

Reply:
https://github.com/sindresorhus/del/issues/32